views/src/app/Auth/AuthorizedApp.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import Nav from './components/navbar/Navbar';
+import Home from './pages/Home/Home';
+import Profile from './pages/Profile/Profile';
+
+function AuthorizedApp({ logoutHandler }) {
+    return (
+        <Router>
+            <div>
+                <Nav logoutHandler={logoutHandler} />
+                <Switch>
+                    <Route path="/profile">
+                        <Profile />
+                    </Route>
+                    <Route path="/">
+                        <Home />
+                    </Route>
+                </Switch>
```

Comment:
Does this mean that an authorized user can't see posts? They can only interact with their profiles?

Reply:
Great catch!
I currently plan to add that later after the post editing pull request (PR #3) - otherwise there is nothing to test it on!